### ISSUE TITLE
Build aborts due to warnings

### ISSUE THREAD

**Issue by MatthewAlner** [chinchang/superplaceholder.js#15]
jshint complains about prettiers wrapping of a ternary operator

```sh
➜  superplaceholder.js git:(master) ✗ grunt build
Running "jshint:files" (jshint) task

   src/superplaceholder.js
     88 |            ? ''
                     ^ Misleading line break before '?'; readers may interpret this as an expression boundary.

>> 1 error in 2 files
Warning: Task "jshint:files" failed. Use --force to continue.

Aborted due to warnings.
```

**Comment by MatthewAlner** [comment]
You could up prettiers "printWidth" from 80 to 120... or just remove jshint and get grunt to run prettier as part of the build step

**Comment by MatthewAlner** [comment]
looks like it might be this issue https://github.com/jshint/jshint/issues/2528 adding "laxbreak" will fix it for now

**Comment by chinchang** [comment]
Fixed in #16
